lib/perl/Genome/Sys.pm

```diff
@@ -1490,24 +1490,30 @@ sub shellcmd {
         $t1 = time();
         my $system_retval;
         eval {
-                my ($restore_stdout);
+            # Use fork/exec here so we can redirect stdout and/or stderr in the child and
+            # not have to worry about resetting them after the child process is done.
+            # Note to the future: FCGI ties STDOUT and STDERR and does something with them
+            # that means you can't open() them with typeglobs or references.  See commit 0d56bbc
+
+            my $pid = fork();
```

Comment:
Maybe?

This process is going away one way or another, either through the exec or by dying.  We don't have to worry about setting up a new UR::Context::Process, or disconnect/reconnect DB handles.